spec/system/task_spec.rb

```diff
@@ -1,53 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project){create(:project)}
+  let!(:task){create(:task, project_id: project.id)}
```

Comment:
associationをFactoryに記載したことで、project_idの指定が不要になるかと思うので確認をお願いします。
```suggestion
  let!(:task){ create(:task) }
```

Reply:
03c0bf8